manim/cli/render/ease_of_access_options.py

```diff
@@ -22,7 +22,7 @@
         "default media player.",
     ),
     option(
-        "-f",
+        "-fb",
```

Comment:
you should give a warning here. possibly `-f` would work like previous versions, with a warning about it going to be changed soom.